nest-clamav-scanner/.env.example

```diff
@@ -4,39 +4,37 @@
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB (Preview).
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 # minio
-MINIO_ACCESS_KEY=acceskey
-MINIO_ENDPOINT=cdn-api.bratislava.sk
-MINIO_HOST=cdn-api.bratislava.sk:443
+MINIO_ACCESS_KEY=forms-dev
+MINIO_ENDPOINT=s3.bratislava.sk
+MINIO_HOST=s3.bratislava.sk:443
 MINIO_PORT=443
-MINIO_SECRET_KEY=secretkey
+MINIO_SECRET_KEY=secret
 MINIO_USE_SSL=true
 
 # clamav
 CLAMAV_HOST=localhost
 CLAMAV_PORT=3310
-CLAMAV_UNSCANNED_BUCKET=calmav-unscanned-bucket
-CLAMAV_SAFE_BUCKET=calmav-clean-bucket
-CLAMAV_INFECTED_BUCKET=calmav-quarantined-bucket
+CLAMAV_UNSCANNED_BUCKET=forms-dev-uploaded
+CLAMAV_SAFE_BUCKET=forms-dev-safe
+CLAMAV_INFECTED_BUCKET=forms-dev-infected
 
 # postgres
-POSTGRES_USER=postgres
-POSTGRES_PASSWORD=postgres
-POSTGRES_DB=postgres
-DATABASE_URL="postgresql://user:pass@localhost:5432/mydb?connect_timeout=30&schema=public"
+DATABASE_URL="postgresql://user:pass@localhost:54302/mydb?connect_timeout=30&schema=public"
 
 # other settings
-PORT=3000
+PORT=3200
 NODE_ENV=development
-NEST_FORMS_BACKEND=nest-forms-backend-url
+NEST_FORMS_BACKEND=http://localhost:3100
 MIMETYPE_WHITELIST="text/csv image/vnd.dwg image/vnd.dxf image/jpeg image/jpg image/png image/tiff image/bmp image/gif application/pdf application/msword application/vnd.openxmlformats-officedocument.wordprocessingml.document application/vnd.ms-excel application/vnd.openxmlformats-officedocument.spreadsheetml.sheet application/vnd.ms-powerpoint application/vnd.openxmlformats-officedocument.presentationml.presentation application/zip application/x-zip-compressed multipart/x-zip"
```

Comment:
We have this copy pasted in at least 3 packages, we need to sync it somehow in the future.